### PR TITLE
update to 0.5.0

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

**Comment by jreback** [comment]
cc @jakirkham

does this need to be rerendered:https://circleci.com/gh/conda-forge/patsy-feedstock/14?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link ?

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by jreback** [comment]
@conda-forge-admin, please rerender

**Comment by jreback** [comment]
@ocefpaf can you re-render again? I don't think mine worked

**Comment by ocefpaf** [comment]
@jreback it takes a while for the bot to process it but https://github.com/conda-forge/patsy-feedstock/pull/7/commits/9f20b9db72def82fdcc2eec48b44e8ce920db076 seems to be OK. Let's wait for the CIs now.

**Comment by jreback** [comment]
ahh ok, thanks!

yeah this is a pretty old recipe

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,9 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ pkg_name }}-{{ version }}.zip
```
Comment (ocefpaf) [inline recipe/meta.yaml:10]: @jreback I removed the `fn`. It is better to let `conda` find the file name from the URL 99.99% of time (rare cases need the `fn`).

**Comment by jreback** [comment]
thanks @ocefpaf

**Pull request merged**